spec/hosts/roles/auxtel_archiver_spec.rb

```diff
@@ -25,7 +25,6 @@
           it { is_expected.to compile.with_all_deps }
 
           include_examples 'common', os_facts: os_facts, site: site
-          include_examples 'lhn sysctls'
```

Comment:
The shared examples also need to be removed from `comcam_archiver_spec.rb`

Reply:
yes, thanks! I was boarding so I had to close the laptop but I saw the rake fail on my cellphone, fixing it right now